public/app/models/query/Group.js

```diff
@@ -0,0 +1,90 @@
+// Based on http://kindohm.github.io/knockout-query-builder/js/group.js
+(function () {
+    'use strict';
+
+    define(
+        [
+            'lodash',
+            'knockout',
+            'models/query/Condition'
+        ],
+        function (_, ko, Condition) {
+            var Group;
+
+            Group = function (queryBuilder, parentGroup) {
```

Comment:
The following 2 lines define the 2 parts of a group - the child rules (conditions and groups) and the operator (and/or).